ci/gpu/build.sh

```diff
@@ -53,37 +55,27 @@ nvidia-smi
 gpuci_logger "Activate conda env"
 . /opt/conda/etc/profile.d/conda.sh
 conda activate rapids
-gpuci_mamba_retry install -y \
-               "cudf=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
-               "cugraph=$MINOR_VERSION.*" \
-               "cuspatial=$MINOR_VERSION.*" \
-               "dask-cudf=$MINOR_VERSION.*" "dask-cuda=$MINOR_VERSION.*" \
-               "numba>=0.54" \
-               "bokeh>=2.4.2,<=2.5" \
-               "rapids-build-env=$MINOR_VERSION.*" \
-               "rapids-notebook-env=$MINOR_VERSION.*"
-
-# https://docs.rapids.ai/maintainers/depmgmt/ 
-# conda remove --force rapids-build-env rapids-notebook-env
-# conda install "your-pkg=1.0.0"
 
 gpuci_logger "Check versions"
 python --version
 $CC --version
 $CXX --version
+
 conda info
 conda config --show-sources
 conda list --show-channel-urls
 
 ################################################################################
-# BUILD - Build cuxfilter from source
+# BUILD - Build cuxfilter
 ################################################################################
 
-gpuci_logger "Build cuxfilter"
-"$WORKSPACE/build.sh" clean cuxfilter
+gpuci_logger "Build and install cuxfilter"
+cd "${WORKSPACE}"
+gpuci_conda_retry build conda/recipes/cuxfilter --python=$PYTHON
+gpuci_mamba_retry install cuxfilter
```

Comment:
We need to add a `--channel`/`-c` flag here to ensure that `conda` installs `cuxfilter` from the local filesytem.

Reply:
PR updated